lib/form/paragraph3.flow

```diff
@@ -151,16 +179,18 @@ renderParagraph(
 		topLineBaseline, interactiveStyles, rtl, isParagraphMarked
 	);
 
+	maxW = renderParaLinesFn(
```

Comment:
double rendering is bad. do not do it

Reply:
This is done once paragraph is initialized. I will look how to cut down the overhead calling the measuring only here.

Reply:
try to avoid double rendering. it is bad. we should not do it.